### PR TITLE
Add release notes for 2.10.21

### PR DESCRIPTION
**Pull request by danidoni**

### CODE REVIEW

**Review by rubhanazeem — DISMISSED** [review]

File: ReleaseNotes-2.10.21 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+
+Bugfixes
+========
+Frontend:
+ * Update rack to version 2.2.6.4
+   - Fixes CVE-2023-27539 Avoid ReDoS in header parsing
+ * Update rack to version 2.2.6.3
```
[the remark this answers is not in the document]
Comment (danidoni) [reply ReleaseNotes-2.10.21:19]: True, changing

File: ReleaseNotes-2.10.21 [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The dist/README.UPDATERS file has information for people updating
+from a previous OBS release.
+
+
+Bugfixes
+========
+Frontend:
+ * Update rack to version 2.2.6.4
+   - Fixes CVE-2023-27539 Avoid ReDoS in header parsing
```
[the remark this answers is not in the document]
Comment (danidoni) [reply ReleaseNotes-2.10.21]: Clarified

File: ReleaseNotes-2.10.21 (hunk 1 above)
[the remark this answers is not in the document]
Comment (krauselukas) [reply ReleaseNotes-2.10.21]: @danidoni I just meant to remove the line about the rack version, but keeping `- Fixes CVE-2023-27530 Possible DoS Vulnerability in Multipart MIME parsing.`

[the remark this answers is not in the document]
Comment (danidoni) [reply ReleaseNotes-2.10.21]: Yeah, sorry. I brought that back.

**Review by bmwiedemann — APPROVED** [review]

**Pull request merged**